src/provider.ts

```diff
@@ -49,4 +51,12 @@ export default class Provider {
       return null;
     }
   }
+
+  /**
+   * Sets the httpClient's default timeout time
+   * @param {number} time
```

Comment:
Could you also add to the comment if this is in seconds or milliseconds? And maybe rename the function to 'setDefaultTimeoutMs' if it's in milliseconds?

Reply:
Looks better! Done